src/pages/HomePage/AuthForms/Login.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import TextField from "@material-ui/core/TextField";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Checkbox from "@material-ui/core/Checkbox";
+import Typography from "@material-ui/core/Typography";
+import Container from "@material-ui/core/Container";
+import { useFormik } from 'formik';
+
+const validate = (values: any)=> {
```

Comment:
Do not use `any`

Reply:
I am sure that formik shares the type information about this function? https://formik.org/docs/guides/typescript